spring-cloud-stream-tuple/src/main/java/org/springframework/cloud/stream/tuple/DefaultTuple.java

```diff
@@ -36,22 +36,23 @@
 import org.springframework.util.ClassUtils;
 
 /**
- * Default implementation of Tuple interface
+ * Default implementation of Tuple interface.
+ *
  * @author Mark Pollack
  * @author David Turanski
  * @author Michael Minella
  */
 public class DefaultTuple implements Tuple {
 
-	private List<String> names;
+	protected List<String> names;
 
-	private List<Object> values;
+	protected List<Object> values;
 
-	private transient ConfigurableConversionService configurableConversionService;
+	protected transient ConfigurableConversionService configurableConversionService;
 
-	private transient Converter<Tuple, String> tupleToStringConverter = new TupleToJsonStringConverter();
+	protected transient Converter<Tuple, String> tupleToStringConverter = new TupleToJsonStringConverter();
 
-	public DefaultTuple(List<String> names, List<Object> values, ConfigurableConversionService 
+	/*package*/ DefaultTuple(List<String> names, List<Object> values, ConfigurableConversionService
```

Comment:
Why change to package level protection?  Was thinking the hard enforcement of not having a ctor vs. only builder was a bit too restrictive.